### ISSUE TITLE
Gives a warning when an Image Style is set in avatarStyle for a ListItem component

### ISSUE THREAD

**Issue by cihadturhan** [react-native-training/react-native-elements#797]
### Description

If you use an image style in `avatarStyle` in a `ListItem`, such as:
```
<ListItem
  title={'Title'} 
  avatar={myImage}
  avatarStyle={{ width: 22, height: 22, resizeMode : 'contain' }}
/>
```
It gives a warning of 
> Failed prop type: Invalid props.avatarStyle key `resizeMode` supplied ListItem

### Solution

I'm about to submit a PR related to this issue.

**Comment by iRoachie** [comment]
Fixed in https://github.com/react-native-training/react-native-elements/commit/881718c18c6f2eaa6e958086da1f20cdd9353732. Will be available in next release

**Comment by xavier-villelegier** [comment]
Available in the `v0.19.0` !
